### ISSUE TITLE
Possible to deploy on Cloudflare Pages?

### ISSUE THREAD

**Issue by mirrorleap** [primodotso/primo-server#26]
Bravo on this awesome project and the inspiration behind it!

Forked the repo, and followed your deployment instructions for Vercel to try to deploy to Cloudflare Pages instead. However, the deployment kept failing for various reasons even after tweaking build settings. Do you think it's possible to deploy or changes are needed to the codebase? 

Looking at the Cloudflare Pages docs and also their migration guide from Vercel, deploying a Svelte/SvelteKit app shouldn't be too hard. However, I have not been able to do this successfully. Or perhaps it's due to Vite.

Could you please shed some light on the issue? Appreciate your help.

**Comment by mateomorris** [comment]
Hey thanks!

Yeah shouldn't be too hard to deploy on Cloudflare Pages instead - I'll take a look tonight or tomorrow to see what's up.

**Comment by mirrorleap** [comment]
Hey, thanks for the super quick reply! Will wait for your analysis.

**Comment by mateomorris** [comment]
@mirrorleap okay you should be able to deploy to Cloudflare Pages now - just needed to swap out the cryptography and buffer libraries. 

 I added instructions to the readme if you need them - you should just need to sync your fork and add a `NODE VERSION = 16` environment variable. Hope you enjoy using it! Let me know if you run into anything else.

**Comment by mirrorleap** [comment]
Hi, thanks for the quick code changes to make it compatible with Cloudflare. Now, the build is fine mostly but I have run into another issue. However, this seems to be a limitation for Cloudflare Workers free tier - https://developers.cloudflare.com/workers/platform/limits/#worker-size

`22:45:00.642 | ✨ Upload complete!
22:45:04.701 | Success: Assets published!
22:45:06.871 | Error: Failed to publish your Function. Got error: Your Functions script is over the 1 MiB size limit (workers.api.error.script_too_large)`

**Comment by mateomorris** [comment]
Interesting. I was able to publish on Cloudflare Pages but I didn't look at the logs or test much - does this happen when you try to do something in particular? I'm guessing SvelteKit publishes its endpoints as Workers?

**Comment by mirrorleap** [comment]
I think it happens just at the end of the build/publish step, and as a result the entire deployment fails. I wasn't actually able to use Primo. Tried a few times, but it kept failing with the same result. Sort of weird as I am using the exact build config  you have documented. 

On a quick web search, I found other users having this same issue with Cloudflare Pages/Workers deploying Next based projects - https://github.com/cloudflare/next-on-pages/issues/68. And the error seems to be due to limitations with the free tier.
